docs/src/Touchable.doc.js

```diff
@@ -159,4 +159,49 @@ card(
   />
```

Comment:
@mminute Can we add this to the `PropTable`?

Reply:
@christianvuerings done!